[2024-01-17 07:33:56,021] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2024-01-16T00:00:00+00:00 [queued]>
[2024-01-17 07:33:56,055] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2024-01-16T00:00:00+00:00 [queued]>
[2024-01-17 07:33:56,056] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-01-17 07:33:56,056] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2024-01-17 07:33:56,056] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-01-17 07:33:56,123] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): also_run_this> on 2024-01-16 00:00:00+00:00
[2024-01-17 07:33:56,152] {standard_task_runner.py:52} INFO - Started process 2271 to run task
[2024-01-17 07:33:56,158] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2024-01-16T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpd2jtmf1o', '--error-file', '/tmp/tmpluj9nuyz']
[2024-01-17 07:33:56,159] {standard_task_runner.py:80} INFO - Job 4: Subtask also_run_this
[2024-01-17 07:33:56,222] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2024-01-17 07:33:56,513] {task_command.py:371} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2024-01-16T00:00:00+00:00 [running]> on host 89a709ccad2d
[2024-01-17 07:33:57,037] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2024-01-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-16T00:00:00+00:00
[2024-01-17 07:33:57,052] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-01-17 07:33:57,053] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2024-01-16T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2024-01-16 00:00:00+00:00: scheduled__2024-01-16T00:00:00+00:00, externally triggered: False>"']
[2024-01-17 07:33:57,171] {subprocess.py:85} INFO - Output:
[2024-01-17 07:33:57,172] {subprocess.py:92} INFO - run_id=scheduled__2024-01-16T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2024-01-16 00:00:00+00:00: scheduled__2024-01-16T00:00:00+00:00, externally triggered: False>
[2024-01-17 07:33:57,172] {subprocess.py:96} INFO - Command exited with return code 0
[2024-01-17 07:33:57,517] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20240116T000000, start_date=20240117T073356, end_date=20240117T073357
[2024-01-17 07:33:57,592] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-17 07:33:57,714] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
